.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+# This is a the GitHub Action to run the CI for SODA Dashboard
+
+name: SODA Dashboard CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ master ]
+      
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-16.04]
```

Comment:
can we also add latest ubuntu OS? ( suggestion)

Reply:
Added and updated the PR. @asifdxtreme